File: Rlm.Core/Model/Standing.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Rlm.Core
{
    public interface IStanding
    {
        ICar Car { get; }
        int Points { get; }
        string Position { get; }
        long AverageTime { get; }
    }

    public class Standing : IStanding
    {
        public ICar Car { get; set; }
        public int Points { get; set; }
        public string Position { get; set; }
        public long AverageTime { get; set; }
    }
}

File: mobile-clients-routing/src/app/models/standing.ts
import { Car } from './car';

export class Standing {
    car: Car;
    position: string;
    points: number;
    averageTime: number;
    averageTimeSeconds: number;

    static fromPayload(payload: any): Standing {
        return {
            car: Car.fromPayload(payload.Car),
            position: payload.Position,
            points: payload.Points,
            averageTime: payload.AverageTime,
            averageTimeSeconds: payload.AverageTime / 1000
        };
    }
    
    static getStandingsFromPayload(payload: any): Array<Standing> {
        let standings: Array<Standing> = new Array<Standing>();
        
        for (let s of payload) {
            standings.push(Standing.fromPayload(s));
        }

        return standings;
    }
}